File: ATS.UI/ClientApp/src/app/core/interfaces/reviewQueue.ts
export class IReviewQueueDisplay {
    reviewQueueID: number;
    requirementID: number;
    requirement: string;
    candidateID: number;
    companyID: number;
    candidateCompanyID: number;
    candidate: string;
    rate: number;
    rateTypeID: number;
    rateType: string;
    submittedBy: number;
    submittedByName: string;
    dateSubmitted: Date;
    reviewerID: number;
    reviewer: string;
    dateReviewed: Date;
    reviewStatusID: number;
    reviewStatus: string;
    recordID?: number;
    dateCreated: Date;
    createdBy: string;
    dateModified: Date;
    modifiedBy: string;
    timeStamp: Date;
    requirementCandidate?: string;
    fileName: string;
    resumeURL: string;
    note?: string;
    workAuth?: string;
}

export class IReviewQueueForm {
    ReviewQueueID: number;
    RequirementID: number;
    CandidateID: number;
    Rate?: number;
    RateTypeID?: number;
    SubmittedBy: number;
    DateSubmitted: Date;
    ReviewerID?: number;
    DateReviewed?: Date;
    ReviewStatusID: number;
    Note: string;
    //recordID?: number;
    DateCreated: Date;
    CreatedBy: string;
    DateModified: Date;
    ModifiedBy: string;
    //TimeStamp: Date;
}

File: ATS.DTO/ReviewQueue.cs
using System;

namespace ATS.DTO
{
    public class ReviewQueue : AbstractDataObject
    {
        public int ReviewQueueID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }
        public int RequirementID { get; set; }
        public int CandidateID { get; set; }
        public int? CompanyID { get; set; }
        [Ignore]
        public int? CandidateCompanyID { get; set; }
        public int? Rate { get; set; }
        public int? RateTypeID { get; set; }
        public int SubmittedBy { get; set; } // current logged in user ID
        public DateTime? DateSubmitted { get; set; }
        public int? ReviewerID { get; set; }
        public DateTime? DateReviewed { get; set; }
        public int ReviewStatusID { get; set; }  // 8
        public string Note { get; set; }

        [Ignore]
        public string Requirement { get; set; }
        [Ignore]
        public string Candidate { get; set; }
        [Ignore]
        public string RateType { get; set; }
        [Ignore]
        public string SubmittedByName { get; set; }
        [Ignore]
        public string Reviewer { get; set; }
        [Ignore]
        public string ReviewStatus { get; set; }
        [Ignore]
        public string ResumeURL { get; set; }
        [Ignore]
        public string FileName { get; set; }
        [Ignore]
        public string WorkAuth { get; set; }
    }

}
